MarsRover/MapDisplay.cs

```diff
@@ -2,7 +2,8 @@ namespace MarsRover
 {
     public class MapDisplay
     {
-        public const string Rover = "🔴";
-        public const string Obstacle = "🪨";
+        public const string Rover = "👹";

```

Comment:
What does the `Rover` look like when it's facing `Direction.North` vs `Direction.South`? Or is it the same?